### DISCUSSION TITLE
Can I join a people's organization for my studies ?

### DISCUSSION THREAD

**Discussion by ngogiaphat** [nuxt/nuxt#23728] [General]
The organizations I want to join to study are:
1/VueJS
2/NuxtLab
3/Nuxt
4/Vueuse
5/Vite
6/Vitest
7/Nuxt Modules
8/UnoCSS
9/UnJS

i just read the code, read-only =(((( I want to learn them, pls acp me to the organization, pls =(((((
thank u, thanks everyone 🥰

**Comment by warflash** [comment]
Isn't the beauty of open source that everyone can read the code without being part of the org maintaining it? There shouldn't be anything that's stopping you from reading it 🙏
